homeassistant/components/generic/config_flow.py

```diff
@@ -273,8 +280,8 @@ async def async_step_user(
             ):
                 errors["base"] = "no_still_image_or_stream_url"
             else:
-                errors, still_format = await async_test_still(self.hass, user_input)
-                errors = errors | await async_test_stream(self.hass, user_input)
+                errors, still_format = await async_test_still(hass, user_input)
+                errors = errors | await async_test_stream(hass, user_input)
                 still_url = user_input.get(CONF_STILL_IMAGE_URL)
```

Comment:
We don't need to assign these local variables until we've checked for errors on line 288. We can move these lines inside that block below.

Reply:
Done 52b561b